hci/command/commands/hci_ext_packet_error_rate_cmd.py

```diff
@@ -0,0 +1,44 @@
+from struct import pack, unpack
+
+from ..command_packet import CommandPacket
+from enum import IntEnum
+
+
+class HCI_EXT_PacketErrorRateCmd(CommandPacket):
+
+    class Command(IntEnum):
+        HCI_EXT_PER_RESET = 0x00
+        HCI_EXT_PER_READ = 0x01
+
+    def __init__(self, conn_handle, command):
+        super().__init__(
+            CommandPacket.OpCode.HCI_EX_PER,
+            HCI_EXT_PacketErrorRateCmd._params_to_binary(conn_handle, command)
+        )
+
+    @staticmethod
+    def _params_to_binary(conn_handle, client_rx_mtu):
+        return pack('<HH', conn_handle, client_rx_mtu)
+
+    @property
+    def conn_handle(self):
+        OFFSET, SIZE_OCTETS = 4, 2
+        data = self._get_data(OFFSET, SIZE_OCTETS)
+        return unpack('<H', data)[0]
+
+    @property
+    def command(self):
+        OFFSET, SIZE_OCTETS = 6, 2
+        data = self._get_data(OFFSET, SIZE_OCTETS)
+        return unpack('<H', data)[0]
+
+    def __str__(self):
+        return super().__str__() + '\n' + '\n'.join([
+            'Connection Handle: {} ({})',
+            'Command: {} ({})',
+        ]).format(
+            hex(self.conn_handle),
+            int(self.conn_handle),
+            hex(self.command),
+            int(self.command),
```

Comment:
Translate `command` using the `enum` you created.
For example: `HCI_EXT_PacketErrorRateCmd.Command(self.command).name`